auto_mycosnp_ICA.py

```diff
@@ -0,0 +1,166 @@
+        try:
+            tmp= icav_out(bashCommand)
+        except:
+            del ica_target[i]
+
+ica_target=folders[ica_target[0]]
+
+#Mycosnp needs a samplesheet, as input and must be uploaded into ICA
+try:
+    os.mkdir("./%s_samplesheet/" % run_name)
+except:
+    print("Folder Already Exits")
+
+#The names of the samples, and there ICA ids must be downloaded from ICA
+bashCommand = "icav2 projectdata list  --parent-folder " +  ica_target + "output/fastqs/Samples/"
+tmp= icav_out(bashCommand)
+
+ica_lanes=[]
+for i in tmp:
+    ica_lanes.append(i.split(" ")[1])
+
+ica_names=[]
+for i in ica_lanes:
+    bashCommand = "icav2 projectdata list  --parent-folder  " +  ica_target + "output/fastqs/Samples/Lane_1/"
+    tmp= icav_out(bashCommand)
+    for j in tmp:
+        if not "Undetermined" in j.split(" ")[0]:
+            ica_names.append(j.split(" ")[0])
+
+ica_ids=[]
+sample_names=[]
+for i in ica_names:
+    bashCommand = 'icav2 projectdata list  --parent-folder ' + i
+    tmp= icav_out(bashCommand)
+    for j in tmp:
+        ica_ids.append(j.split(" ")[4]+",")
+        sample_names.append(j.split(" ")[1])
+
+
+sample_sheet=[]
+sample_sheet.append('sample_id,fastq_1,fastq_2\n')
+for i in sample_names:
+    if i.split('_')[3] == 'R1':
+        tmp=i.split('_')[0]+','+i
+        tmp=tmp+','+i.replace('R1','R2')+'\n'
+        sample_sheet.append(tmp)
+
+bashCommand = "icav2 projects enter Testing"
+process = subprocess.Popen(bashCommand.split(" ",3), stdout=subprocess.PIPE)
+process.communicate()
+
+file = open("./%s_samplesheet/samplesheet.csv" % run_name, "w")
+file.write("".join(sample_sheet))
+file.close()
+
+file = open("./%s_samplesheet/samples.csv" % run_name, "w")
+file.write("".join(ica_ids))
+file.close()
+
+bashCommand = "icav2 projects enter Testing"
+process = subprocess.Popen(bashCommand.split(" ",3), stdout=subprocess.PIPE)
+process.communicate()
+
+#All samples must be linked to Testing project in order to run in the pipeline
+sams=open('./%s_samplesheet/samples.csv' % run_name,'r').read()
+for i in range(len(sams.split(','))):
+       bashCommand = "icav2 projectdata link %s" % sams.split(',')[i]
+       process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE)
+       process.communicate()
+
+bashCommand = "icav2 projectdata link %s" % ica_target
+process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE)
+process.communicate()
+
+bashCommand = "icav2 projectdata upload ./%s_samplesheet" % run_name
+process = subprocess.Popen(bashCommand.split(" "), stdout=subprocess.PIPE, stdin=subprocess.PIPE)
+process.communicate(input=b'y')
+
+bashCommand = "icav2 projectdata list --data-type FILE --parent-folder /%s_samplesheet/ --file-name samplesheet.csv" % run_name
+tmp= icav_out(bashCommand)
+samplesheet=tmp[0].split(' ')[4]
+
+# Create icav2 argument
+arg = "icav2 projectpipelines start nextflow CDC_mycosnp_V1_3 --input input_samples:"
```

Comment:
Same as the auto_grandeur_ICA.py, I'm debating whether or not `CDC_mycosnp_V1_3` should be a variable.

It probably doesn't matter.